File: PATTERNS/Pattern35/Pattern35.c
#include<stdio.h>

void main(){

    int i, k, n;

    printf("Enter the number: ");
    scanf("%d", &n);

    for (i = 0; i < n; i++) {
        // padding front space
        for (k = n-i-1; k >= 0; k--) {
            printf(" ");
        }
        // left star boarder
        printf("*");
        // inner stars
        for (k = n - 2; k > 0; k--) {
            if (i == 0 || i == (n - 1)) {
                printf("*");
            } else {
                printf(" ");
            }
        }
        // right star boarder
        printf("*\n");
    }
}

File: PATTERNS/Pattern35/Pattern35.go
package main

import "fmt"

func main() {
	var n int
	fmt.Printf("Enter the row number:")
	fmt.Scanln(&n)
	for i := 1; i <= n; i++ {
		for k := n; k >= i; k-- {
			fmt.Print(" ")
		}
		for j := 1; j <= n; j++ {
			if i == 1 || i == n {
				fmt.Print("*")
			} else if j == 1 || j == n {
				fmt.Print("*")
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}

}
